components/director/run.sh

```diff
@@ -303,7 +303,15 @@ export APP_FORMATION_ASSIGNMENT_ASYNC_STATUS_API_ENDPOINT='/{ucl-formation-id}/a
 export APP_FORMATION_ASSIGNMENT_ASYNC_STATUS_RESET_API_ENDPOINT='/{ucl-formation-id}/assignments/{ucl-assignment-id}/status/reset'
 export APP_FORMATION_ASYNC_STATUS_API_ENDPOINT='/{ucl-formation-id}/status'
 export APP_UCL_CERT_OU_SUBACCOUNT_ID='f8075207-1478-4a80-bd26-24a4785a2bfd'
-export APP_TENANT_MAPPING_CONFIG_PATH="/tmp/tenant-mapping-config.json"
+# This file previously was created with the help of the Helm values.yaml
+# However, the chart was moved in the internal repository and the tenant-mappings need to be created manually
+# yq eval ".global.director.tenantMappings" chart/compass/values.yaml  -o=json > /tmp/tenant-mapping-config.json
```

Comment:
you can also add the command as `echo` message in the else clause. It will be easier when the script fails to have the command in the terminal and execute it